src/js/stores/preferences.js

```diff
@@ -99,6 +99,10 @@ define(function (require, exports, module) {
             return this._preferences;
         },
 
+        get: function (key, unsetValue) {
```

Comment:
@iwehrman can you comment on this since you wrote it? I can change it but I wanted to make sure there wasn't a specific reason for your method first.